sitemedia/js/components/books/BooksSearch.js

```diff
@@ -0,0 +1,166 @@
+import { mapGetters, mapActions, mapState, mapMutations } from 'vuex'
+import isEqual from 'lodash/isEmpty'
+
+import SearchFacet from '../SearchFacet'
+import SearchFilter from '../SearchFilter'
+import SearchSort from '../SearchSort'
+import { isEmpty } from '../../../../node_modules/rxjs/operators';
+
+export default Vue.component('BooksSearch', {
+    template: `
+    <div class="books-search">
+        <sui-container textAlign="center" text>
+            <h4 is="sui-header" class="results-count">
+            Displaying {{ totalResults }} {{ totalResults | pluralize('book') }}
+            </h4>
+        </sui-container>
+        <form class="search-form ui form">
+            <sui-menu tabular attached="top">
+                <div
+                    is="sui-menu-item"
+                    v-for="(tab, index) in tabs"
+                    :key="index"
+                    :active="activeTab === index"
+                    :content="tabLabel(tab)"
+                    @click="activeTab = index"
+                />
+            </sui-menu>
+            <sui-segment
+                v-for="(tab, index) in tabs"
+                :key="index"
+                :class="{ active: activeTab === index }"
+                class="bottom attached tab"
+            >
+                <div class="ui equal width stackable grid">
+                    <search-facet
+                        v-for="facetName in tab"
+                        v-bind="facetConfig.find(facet => facet.name === facetName)"
+                        :choices="facetChoices.filter(choice => choice.facet === facetName)"
+                        :key="facetName"
+                    >
+                    </search-facet>
+                </div>
+            </sui-segment>
+            <sui-segment inverted class="active-items">
+                <label>Selected</label>
+                <div class="active-facet-choice" v-for="(choice, index) in activeFacetChoices" :key="index">
+                    <sui-label>
+                        <span class="facet-name">{{ facetConfig.find(facet => facet.name === choice.facet).label }}: </span>
+                        <span class="choice-name">{{ choice.value }}</span>
+                        <sui-icon name="delete" @click="toggleFacetChoice(choice)" />
+                    </sui-label>
+                </div>
+            </sui-segment>
+            <sui-segment inverted>
+                <search-sort></search-sort>
+            </sui-segment>
+        </form>
+    </div>`,
+    components: {
+        SearchFacet,
+        SearchFilter,
+        SearchSort,
+    },
+    data() {
+        return {
+            tabs: [
+                ['author', 'editor', 'translator'],
+                ['publisher', 'pub_year'],
+                ['language', 'subject'],
+                ['annotator']
+            ],
+            activeTab: 0,
+            facetConfig: [
```

Comment:
These would also be great to configure from the Django form somehow (I know we discussed that at some point)

Reply:
Yeah, hoping to use the same approach here.